lib/daru/dataframe.rb

```diff
@@ -117,6 +117,21 @@ def from_plaintext path, fields
         Daru::IO.from_plaintext path, fields
       end
 
+      # Read the table data from a remote html file. For this method to work,
+      # the data should be present in a html file with proper combination of <table>, <th> and <td> tags.
+      #
+      # == Arguments
+      #
+      # * path - Path / URL of the html file to be read.
+      # * fields - A Hash, with match key to search. data, order, index keys can overwrite the scraped values from web.
+      #
+      # == Usage
+      #
+      # df = Daru::DataFrame.from_html(path, index: [:p1, :p2, :p3, :p4, :p5, :p6, :p7])[0]
```

Comment:
And why it ends up with `[0]`? (Well, I understand there could be several tables returned, but it is absolutely not obvious from docs or example).